integration/test/performance_tracker/transmitter/transmitter.go

```diff
@@ -0,0 +1,237 @@
+*/
+func (transmitter * TransmitterAPI) CreateTable() error{
+	input := &dynamodb.CreateTableInput{
+		AttributeDefinitions: []*dynamodb.AttributeDefinition{
+			{
+				AttributeName: aws.String("Hash"),
+				AttributeType: aws.String("S"),
+			},
+		},
+		KeySchema: []*dynamodb.KeySchemaElement{
+			{
+				AttributeName: aws.String("Hash"),
+				KeyType:       aws.String("HASH"),
+			},
+		},
+		ProvisionedThroughput: &dynamodb.ProvisionedThroughput{
+			ReadCapacityUnits:  aws.Int64(10),
+			WriteCapacityUnits: aws.Int64(10),
+		},
+		TableName: aws.String(transmitter.DataBaseName),
+	} // this is the config for the new table
+	
+	_, err := transmitter.DynamoDbClient.CreateTable(input)
+	if err != nil {
+		fmt.Printf("Got error calling CreateTable: %s", err)
+		return err
+	}
+	
+	fmt.Println("Created the table", transmitter.DataBaseName)
+	return nil
+}
+/*
+AddItem() 
+Desc: Takes in a packet and
+will convert to dynamodb format  and upload to dynamodb table.
+Param: 
+	packet * map[string]interface{}:  is a map with data collection data
+Side effects: 
+	Adds an item to dynamodb table
+*/
+func (transmitter * TransmitterAPI) AddItem(packet map[string]interface{})(string,error){
+	// fmt.Printf("Packet: %+v \n",packet)
+	// metrics, _ := dynamodbattribute.MarshalMap(packet.Metrics)
+	// fmt.Printf("Metric: %+v \n OG:%+v \n",metrics, packet.Metrics)
+	item, err := dynamodbattribute.MarshalMap(packet)
+	if err != nil {
+		panic(err)
+	}
+	DBitem := &dynamodb.PutItemInput{
+		Item:      item,
+		TableName: aws.String(transmitter.DataBaseName),
+	}
+	// fmt.Println(DBitem)
+	// return err
+	_, err = transmitter.DynamoDbClient.PutItem(DBitem)
+	if err != nil {
+		fmt.Printf("Couldn't add item to table. Here's why: %v\n", err)
+	}
+	return fmt.Sprintf("%v",item),err
+}
+/*
+TableExist()
+Desc: Checks if the the table exist and returns the value
+//https://github.com/awsdocs/aws-doc-sdk-examples/blob/05a89da8c2f2e40781429a7c34cf2f2b9ae35f89/gov2/dynamodb/actions/table_basics.go
+*/
+func (transmitter * TransmitterAPI) TableExist() (bool,error){
+	l,err := transmitter.ListTables()
+	for i:=0; i< len(l); i++{
+		if transmitter.DataBaseName == l[i]{
+			return true,nil
+		}
+	}
+	return false,err
+
+}
+/*
+RemoveTable()
+Desc: Removes the table that was craeted with initialization. 
+Side effects: Removes a Dynamodb table
+*/
+func (transmitter * TransmitterAPI) RemoveTable() error{
+	input := dynamodb.DeleteTableInput{TableName: aws.String(transmitter.DataBaseName)}
+
+	_,err:=transmitter.DynamoDbClient.DeleteTable(&input)
+	if err !=nil{
+		fmt.Println(err)
+	}
+	return err
+}
+// ListTables lists the DynamoDB table names for the current account.
+func (transmitter * TransmitterAPI) ListTables() ([]string, error) {
+	var tableNames []string
+	input := &dynamodb.ListTablesInput{}
+	tables, err := transmitter.DynamoDbClient.ListTables(
+		input)
+	if err != nil {
+		fmt.Printf("Couldn't list tables. Here's why: %v\n", err)
```

Comment:
"Here's why" is not necessary here. 
```suggestion
		fmt.Printf("Error listing tables: %v\n", err)
```